reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
Doesn't that add another `<link rel=stylesheet` to every page meaning a second request for CSS? Is it possible to get this built into the main LESS output instead?